src/acachecontrol/cache.py

```diff
@@ -25,23 +28,25 @@ def add(self, key, value):
         self.cache[hashable_key] = value
         self.release_new_key(key)
 
-    def get(self, key):
+    def get(self, key: Tuple[str, str, Dict]) -> Any:
         return self.cache.get(self._make_key_hashable(key))
 
-    def delete(self, key):
-        pass
+    def delete(self, key: Tuple[str, str, Dict]) -> None:
+        self.cache.pop(self._make_key_hashable(key), None)
 
-    def clear_cache(self):
+    def clear_cache(self) -> None:
         self.cache = {}
 
-    async def register_new_key(self, key, wait_timeout=10):
+    async def register_new_key(
+        self, key: Tuple[str, str, Dict], wait_timeout: float = 10.0
```

Comment:
We'll need to make wait_timeout configurable, but it is ok for now

Reply:
what do you mean?

Reply:
something like AsyncCacheControl(wait_timeout=120)